docs/docs/user/projects/models/create.md

```diff
@@ -101,6 +101,12 @@ name for the repository and project.
 
 ## Step 4: Metadata
 
-This is an important step. Here, you can select the version and the model
-nature of your tool. If you don't select any version, the functionality will be
-restricted. You will not be able to setup backups or create read-only sessions.
+This is an important step.
+
+Here, you can select the version and the model nature of your tool. If you
+don't select any version, the functionality will be restricted. You will not be
+able to setup backups or create read-only sessions.
+
+For Capella the best version to pick currently is 6.0.0. Older versions are
+outdated and the possibility might to use them might be removed in the future.
+Make sure your model is up to date and you update it in time.
```

Comment:
We should not give any recommendations about specific versions. They become outdated pretty soon.